README.md

```diff
@@ -27,12 +63,13 @@ The governing flow for how tests are executed stem from the `.evergreen/config.y
 
 -   `name` -- This should be in the format test-{pipeline}-{language}-{os}
 -   `display_name` -- This can be named however you see fit. Ensure it is easy to understand. See `.evergreen/config.yml` for examples
--   [`expansions`](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Configuration-Files/#expansions) -- These are how we store buildvariant specific variables. Additionally, for expansions that need to be maintained as secrets, please update those in t[he evergreen project settings](https://spruce.mongodb.com/project/ai-ml-pipeline-testing/settings/variables) using [variables](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-and-Distro-Settings#variables). Some common expansions this takes are:
+-   [`expansions`](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Configuration-Files/#expansions) -- These are how we store buildvariant specific variables. Additionally, for expansions that need to be maintained as secrets, please update those in [the evergreen project settings](https://spruce.mongodb.com/project/ai-ml-pipeline-testing/settings/variables) using [variables](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-and-Distro-Settings#variables). Some common expansions needed are:
     -   `DIR` -- The subdirectory where the tasks will run
     -   `REPO_NAME` -- The name of the repository that will get cloned
     -   `CLONE_URL` -- The github url to clone into the specified DIR
+    -   `DATABASE` -- The database atlas cli will load your index configs (optional)
 
--   `run_on` -- Specified platform to run on. `ubuntu2204-small` is a fine default
+-   `run_on` -- Specified platform to run on. `rhel87-small` is a fine default. Any other distro may fail atlas cli setup.
```

Comment:
```suggestion
-   `run_on` -- Specified platform to run on. `rhel87-small` is a fine default. Any other distro may fail Atlas CLI setup.
```

If any other distro may fail, should we require that `rhel87-small` be used?

Reply:
I didn't want to require the distro in case other languages had problems (similar to how python had issues using rhel90, or Atlas CLI need rhel87+).

Reply:
Maybe specify that for Python integrations `rhel87-small` should be used by default?